js/modules/loki_message_api.js

```diff
@@ -32,109 +34,143 @@ class LokiServer {
       nonce = await callWorker('calcPoW', timestamp, ttl, pubKey, data64, development);
     } catch (err) {
       // Something went horribly wrong
-      // TODO: Handle gracefully
       throw err;
     }
 
-    const options = {
-      url: `${currentNode.url}/store`,
-      type: 'POST',
-      responseType: undefined,
-      timeout: undefined,
-    };
-
-    const fetchOptions = {
-      method: options.type,
-      body: data64,
-      headers: {
-        'X-Loki-pow-nonce': nonce,
-        'X-Loki-timestamp': timestamp.toString(),
-        'X-Loki-ttl': ttl.toString(),
-        'X-Loki-recipient': pubKey,
-      },
-      timeout: options.timeout,
-    };
-
-    let response;
-    try {
-      response = await fetch(options.url, fetchOptions);
-    } catch (e) {
-      log.error(options.type, options.url, 0, 'Error');
-      throw HTTPError('fetch error', 0, e.toString());
-    }
+    const requests = swarmNodes.map(async node => {
+      const options = {
+        url: `${node}${this.messageServerPort}/store`,
```

Comment:
```suggestion
        url: `${node}:${this.messageServerPort}/store`,
```
With the change requested above

Reply:
I think Beau wanted to allow not specifying the port to allow using the default one (80) implicitly.